pkg/metadatastorage/sqlstore/client.go

```diff
@@ -57,6 +58,33 @@ func ClientWithConnMaxLifetime(connMaxLifetime time.Duration) ClientOption {
 	}
 }
 
+// ensureMySQLConnectionString ensures the connection string has the tcp protocol as required by the go-sql-driver
```

Comment:
Do you have any documentation that this method of wrapping with `tcp()` is required?